src/tools/ts-generator/main.cpp

```diff
@@ -80,34 +75,15 @@ int main(int argc, char* argv[])
     if (! checkOptions(settings))
         return 1;
 
+    // ============ THERMAL : Getting data for generating time-series =========
     auto study = std::make_shared<Data::Study>(true);
     Data::StudyLoadOptions studyOptions;
-    studyOptions.prepareOutput = true;
```

Comment:
`studyOptions.prepareOutput = true;` : disappeared.
In the context of running the **solver**, **prepareOutput** is set as a command line option, and is used to allow / disable / restrain simulation output results.
In the context of running the TS generator, the user wants to generate and print TS. 
He/she doesn't want to know about **prepareOutput**.